README.md

```diff
@@ -41,6 +44,99 @@ This plugin can be consumed by the CAP application deployed on BTP to store thei
 >
 > A reference to adding this can be found [here](https://github.com/cap-java/sdm/blob/691c329f4c3c17ae390cfcb2db1ef02650585aee/cap-notebook/demoapp/pom.xml#L20)
 
+## Setup
+
+In this guide, we use the Bookshop reference sample app in the [deploy branch](https://github.com/cap-java/sdm/tree/deploy) of this repository, to integrate SDM CAP plugin.
+
+### Using the released version
```

Comment:
I see that the steps for released version and development version are same. For released version, we should add the steps of adding the pom.xml dependency. Steps 1-6 should be changed.

Reply:
Done